tests/expectations/snapshots/snap_test_expectation_atomic_renderers.py

```diff
@@ -1025,11 +1004,14 @@
     "name": "atomic.prescriptive.summary",
     "value": {
         "params": {
-            "column_list": {"schema": {"type": "array"}, "value": ["a", "b", "c"]},
+            "column_set": {"schema": {"type": "array"}, "value": ["a", "b", "c"]},
+            "column_set_0": {"schema": {"type": "string"}, "value": "a"},
+            "column_set_1": {"schema": {"type": "string"}, "value": "b"},
+            "column_set_2": {"schema": {"type": "string"}, "value": "c"},
```

Comment:
Another bug. `column_list` params are missing. Also, there is no reason to change the kwarg name from `column_set` to `column_list` in the renderer. This is not a breaking change since the substitution logic is still the same.